### PR TITLE
Add id property to credential-0 credentialSubject

### PR DESCRIPTION
**Pull request by clehner**
Address #46

### CODE REVIEW

File: data/credentials/credential-0.json [hunk 1]
```diff
@@ -5,6 +5,8 @@
   ],
   "type": ["VerifiableCredential"],
   "issuer": "did:example:123",
-  "issuanceDate": "2021-01-01T19:23:24Z",
-  "credentialSubject": {}
+  "issuanceDate": "2022-03-14T21:40:24Z",
+  "credentialSubject": {
+    "id": "did:example:123"
```
Comment (OR13) [inline data/credentials/credential-0.json:10]: Per the spec, `id` is not required... lets add a property that preserves this test case, but that does not leave us with an an empty object.

Comment (OR13) [reply data/credentials/credential-0.json:10]: I suggest we add `{ "@vocab": "https://vocab.example#undefined-term" }` to the context, "foo":"bar" to the subject.

**Review by OR13 — CHANGES_REQUESTED** [review]
See comments

File: data/credentials/credential-0.json (hunk 1 above)
Comment (clehner) [reply data/credentials/credential-0.json]: Amended to use `"foo":"bar"` instead of `id`. Added `{ "@vocab": "https://example.com/#" }` for consistency with the other credentials. Does that work?

**Comment by OR13** [comment]
@clehner merge any PR that I approve at your lesure
